renovate.json

```
@@ -18,8 +18,8 @@
         "packageNames": ["traefik"]
       },
       {
-        "extends": ["schedule:weekly"],
         "packagePatterns": ["circleci/.+"],
+        "schedule": ["after 4am and before 8am on Monday"],
```

I am not completely sure which syntax Renovate follows. Looking at the FAQ, they say they are using Later. Looking at Later's documentation http://bunkat.github.io/later/parsers.html#text it seems like the rule should be:

```suggestion
        "schedule": ["after 4am before 8am on Monday"],
```

---

Compare also our other schedule:

```json
{
"schedule": ["after 8:00 before 20:00 every weekday"]
}
```

---

Good catch! Thank you for validating this so thoroughly. 🙏

---

Updated the commit ([diff](https://github.com/PicturePipe/renovate-config/compare/c836e27387dea4f84668fc89b225002ba73fc348..7bf333f39b477dab0cb8ccada434f9b87ad727a9)).